nixos/modules/services/networking/networkmanager.nix

```diff
@@ -1,40 +1,85 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
+  inherit (builtins)
+    isInt
+    isBool
+    isString toString
+    concatStringsSep
+    attrNames
+    listToAttrs
+    ;
+
+  inherit (lib)
```

Comment:
Most of these occur just a few times, it would be cleaner to just use `lib.` prefix inline. Same for the `builtins`.

Reply:
Just for the ones that don't get used often, or for every invocation of a function in `lib`? If it's the former, then what's the threshold?